blog/_posts/btcpay-server-1-4-0.md

```diff
@@ -0,0 +1,75 @@
+We hope you like these chages, and if you have any feedback feel free to reach out in our [community chat](https://chat.btcpayserver.org/), we'll be glad to have you.
+
+### **Greenfield** 🔌
+
+Our Greenfield API, as usual, also gets a few additions and bug fixes in this update.
+Namely:
+* Adds a `missingPermission` field to 403 errors, to make it easier to identfy why users are getting this error
+* Adds a field `StoreId` to the payment requests
+* Users can now create payment requests through Greenfield without specifying a currency in which case the payment request currency will be set to the store's default currency
+* Improves permission error messages of Greenfidl API
+* Removes redundant/unused parameters in payment methods
+* Fixes a payment request creation failure when expiry was specified
+* Fixes guests not being able to change payment methods when using the internal LN node
+* Fixes `created` field in payment requests to be a UNIX timestamp
+* Fixes `CanViewStoreSettings` permission to enable fetching a fee rate
+
+### **System upgrades** 💻
+
+In 1.4.0, BTCPay Server also upgrades some if it's backend utilities:
+* Migrates from .NET 3.1 to .NET 6.0
+* Can now use more modern C# 10.0 features instead of older constructs
+* Cleans up 4 years of incremental changes, resulting in Class and Route renaming
+
+### **Misc.**
+
+1.4.0 also includes minor changes, bug fixes and improvements all around the product, for example:
+* Supports new TLS version of the SMTP server
+* Fixes sending two emails for each invoice expiry, payment of confirmation/completion
+* Changes the UI terminology for invoices to Processing/Settled, instead of Paid/Confirmed/Complete
+
+The full list of these minor changes can be viewed in the [Changelog](https://github.com/btcpayserver/btcpayserver/releases)
+
+----
+
+That's going to be all for highlights of this BTCPay Server 1.4.0 release! 
+It is now time for us to wish everyone the very best for the year 2022! 💚
+
+As usual, if you have problems, feedback, feature requests regarding BTCPay Server, feel free to reach out on our [community chat](https://chat.btcpayserver.org/). We hope you enjoy what this update has to offer.
+
+As always, thank you to our invaluable contributors, we love you:
+
+[@andhans](https://twitter.com/andhans_jail) [@britttttk](https://github.com/britttttk) [@dennisreimann](https://github.com/dennisreimann) [@dstrukt](https://github.com/dstrukt) [@Kukks](https://github.com/kukks/) [@NicolasDorier](https://github.com/nicolasdorier/) [@Pavlenex](https://github.com/pavlenex/) [@rockstardev](https://github.com/rockstardev/) [@ubolator](https://github.com/bolatovumar) [@woutersamaey](https://github.com/woutersamaey) [@Zaxounette](https://github.com/zaxounette)
```

Comment:
Are you sure you've listed everyone who contributed to this release? I don't think andhans was involved in this release for example?

Reply:
```suggestion
[@britttttk](https://github.com/britttttk) [@dennisreimann](https://github.com/dennisreimann) [@dstrukt](https://github.com/dstrukt) [@Kukks](https://github.com/kukks/) [@NicolasDorier](https://github.com/nicolasdorier/) [@Pavlenex](https://github.com/pavlenex/) [@rockstardev](https://github.com/rockstardev/) [@ubolator](https://github.com/bolatovumar) [@woutersamaey](https://github.com/woutersamaey) [@Zaxounette](https://github.com/zaxounette)
```

I checked against the names in the Changelog. Forgot to take out Andhans.
Left in Core contributors like Rockstar and Britt, even though they don't appear in the Changelog.